### PR TITLE
Fixed a missing damage value

### PR DESCRIPTION
**Pull request by Scischm**
#### Summary
SUMMARY: Bugfixes "Adds a damage value to repeating crossbows." 

#### Purpose of change
Repeating crossbows had no ranged damage value for the new bolt proportional damage changes to scale off of.

#### Describe the solution
This simply adds a ranged damage value to the repeating crossbow. The chosen damage value of 12 is based off half the difference between the damage of the pistol crossbow (7) and the regular crossbow (17).

### CODE REVIEW

**Comment by KorGgenT** [comment]
you need to remove the square brackets from your summary. then type "jenkins rebuild" in as a comment to this pr to reload gorgon.

**Comment by Scischm** [comment]
jenkins rebuild

**Comment by kevingranade** [comment]
I'm not sure about the damage value, the pistol crossbow and repeating crossbow are pretty different in how they function.
A typical pistol crossbow has a draw rate around 80lbs, a repeating crossbow seems to have a draw weight around 25-40lbs.

**Comment by Scischm** [comment]
> A typical pistol crossbow has a draw rate around 80lbs, a repeating crossbow seems to have a draw weight around 25-40lbs.

There is a huge difference between draw weights of modern crossbows with some topping over 200 lbs of draw weight. The regular crossbow has a damage value of 17, the compound a damage of 20. This brings into question how much the draw weight contributes to damage points in game. If we follow a direct approach of the repeating crossbow having roughly half the draw weight of the pistol crossbow that would put its damage at around a pitiful 3 damage per shot even with decent bolts. 

If we applied this same approach to the damage of the the regular crossbow, speculating that an extra 40 pounds of draw weight is a safe middle ground between the 80 lbs of the pistol crossbow and 120ish pounds of draw (which is pretty respectable) that would only add roughly 3 damage to the existing 7 damage of the pistol crossbow which only puts us at 10 damage. Even if we upped the draw 40 more lbs following this method for 160 pounds of draw which is pushing the higher end of the crossbow draw weight spectrum, that would put us at another 3 points of damage higher at 13 damage which is still nowhere near the 17 damage that the regular crossbow puts out. 

If we continued and hit 200 pounds of draw weight (which i would expect from the compound crossbow, not the regular crossbow) that would put us at 16 damage which STILL doesn't meet the 17 damage of the regular crossbow.

TL;DR  How should we approach draw weight vs damage?  Also, should the crossbow pistol be buffed if it has a pretty decent draw weight of 80 lbs but only 7 damage?

From my personal experience, I have played a fair bit with a damage value of 11 for the repeating crossbow and it felt relatively balanced in terms of the new changes with bolt proportional damage. I feel that a damage of 3 per shot would leave the repeating crossbow wanting, but that is obviously still better than it being completely unusable (like it currently is doing 0 damage).
